contentcuration/contentcuration/management/commands/mark_incomplete.py

```diff
@@ -13,49 +13,108 @@
 from contentcuration.models import ContentNode
 from contentcuration.models import File
 
-logmodule.basicConfig()
+logmodule.basicConfig(level=logmodule.INFO)
```

Comment:
Before this line, you'll need to add this line to get `progressbar` working:

`progressbar.streams.wrap_stderr()`

See https://progressbar-2.readthedocs.io/en/latest/#progressbars-with-logging